app/models/cqm/measure.rb

```diff
@@ -10,8 +10,6 @@ class Measure
     validates_inclusion_of :measure_scoring, in: %w[PROPORTION RATIO CONTINUOUS_VARIABLE COHORT]
     validates_inclusion_of :calculation_method, in: %w[PATIENT EPISODE_OF_CARE]
 
-    TYPES = %w[ep eh].freeze
```

Comment:
Why was this removed?

Reply:
This was leftover from the old model. It isn't used anymore.